keps/sig-cluster-lifecycle/draft-20180119-cluster-api.md

```diff
@@ -0,0 +1,241 @@
+# Kubernetes Cluster Management API
+
+## Metadata
+```
+---
+kep-number: draft-20180119
+title: Kubernetes Cluster Management API
+status: Draft
+authors:
+  - name: Robert Bailey
+    github: roberthbailey
+    email: robertbailey@google.com
+    slack: roberthbailey
+  - name: Jacob Beacham
+    github: pipejakob
+    slack: pipejakob
+owning-sig: sig-cluster-lifecycle
```

Comment:
Ok. I've dropped Brian and moved Tim to a reviewer instead of approver. Not sure who to make an approver so I added myself for now, but that means I'm the author, editor, and approver, which is a bit weird. I'll ask folks at the sig call this week if anyone wants to have their name added.

Reply:
Feel free to list me as approver if you like.  Or someone else from cluster-lifecycle that has some context.